pkg/common/common.go

```diff
@@ -54,6 +54,11 @@ const (
 	Cilium  = "cilium"
 	Kubeovn = "kubeovn"
 
+	Calico_Multus  = "calico-multus"
+	Flannel_Multus = "flannel-multus"
+	Cilium_Multus  = "cilium-multus"
+	Kubeovn_Multus = "kubeovn-multus"
```

Comment:
I think these consts can also be deleted.

Reply:
sorry